resource-management/pkg/aggregrator/aggregator.go

```diff
@@ -101,14 +102,22 @@ func (a *Aggregator) Run() (err error) {
 			c := a.createClient(a.urls[i])
 
 			klog.V(3).Infof("Starting loop pulling nodes from region: %v", a.urls[i])
+
+			var completeDuration time.Duration
```

Comment:
Instead of declaring completeDuration outside of for loop and carrying the value across loop, you can move sleep to the end of the for loop.

Reply:
made the changes